src/Store/Store.ts

```diff
@@ -0,0 +1,51 @@
+import EventBus from "../services/EventBus";
+
+export type TState = Record<string, any>;
+
+export enum StoreEvents {
+	Updated = 'updated',
+}
```

Comment:
Я бы советовал использовать union вместе enum. [Тут](https://stackoverflow.com/questions/40275832/typescript-has-unions-so-are-enums-redundant) хорошо объяснено почему
```suggestion
export type StoreEvents =
  | 'updated'
  | 'any other event'
```